test/base.js

```diff
@@ -139,6 +139,41 @@ class Test {
       t.end()
     })
 
+    test('should get 2nd sheet if sheet ID specified', async (t) => {
+      const spreadsheetId = '10WDbAPAY7Xl5DT36VuMheTPTTpqx9x0C5sDCnh4BGps'
+      const sheetInfo = { sheetId: '784337977' }
+      const result = await this.parser.parse(spreadsheetId, sheetInfo)
+      const expected = [{ a: 10, b: 20, c: 30 }, { a: 40, b: 50, c: 60 }, { a: 70, b: 80, c: 90 }]
+      t.deepEqual(result, expected)
+      t.end()
+    })
+
+    test('should parse properly after changing sheetId at runtime', async (t) => {
+      const spreadsheetId = '10WDbAPAY7Xl5DT36VuMheTPTTpqx9x0C5sDCnh4BGps'
+      const firstSheetId = '0'
+      const secondSheetId = '784337977'
+
+      const firstResult = await this.parser.parse(spreadsheetId, { sheetId: firstSheetId })
+      const firstExpected = [{ a: 1, b: 2, c: 3 }, { a: 4, b: 5, c: 6 }, { a: 7, b: 8, c: 9 }]
+      t.deepEqual(firstResult, firstExpected)
+
+      const secondResult = await this.parser.parse(spreadsheetId, { sheetId: secondSheetId })
+      const secondExpected = [{ a: 10, b: 20, c: 30 }, { a: 40, b: 50, c: 60 }, { a: 70, b: 80, c: 90 }]
+      t.deepEqual(secondResult, secondExpected)
+      t.end()
+    })
+
+    test('should parse properly if both sheetName and sheetId are given', async (t) => {
+      const spreadsheetId = '10WDbAPAY7Xl5DT36VuMheTPTTpqx9x0C5sDCnh4BGps'
+      const sheetName = 'Sheet2'
+      const sheetId = '784337977'
+
+      const result = await this.parser.parse(spreadsheetId, { sheetId: sheetId, sheetName: sheetName })
```

Comment:
You can use property shorthand like below:

`const result = await this.parser.parse(spreadsheetId, { sheetId, sheetName })`